### PR TITLE
updating to new test format for social/storage

### PR DESCRIPTION
**Pull request by ryscheng**

### CODE REVIEW

File: spec/provider.integration.spec.js [hunk 1]
```diff
@@ -11,6 +11,20 @@ var setup = function() {
...
+  .bind(this, window.freedom, "scripts/providers/social/websocket-server/social.ws.json", {}));
+describe("integration-double: social.ws.json", require("freedom/spec/providers/social/social.double.integration.src")
+  .bind(this, window.freedom, "scripts/providers/social/websocket-server/social.ws.json", {}));
+
+// Storage
+describe("integration: storage.isolated.json", require("freedom/spec/providers/storage/storage.integration.src")
+  .bind(this, window.freedom, "scripts/providers/storage/isolated/storage.isolated.json", {}, false));
+describe("integration: storage.shared.json", require("freedom/spec/providers/storage/storage.integration.src")
+  .bind(this, window.freedom, "scripts/providers/storage/shared/storage.shared.json", {}, false));
+/**
```
Comment (willscott) [inline spec/provider.integration.spec.js]: remove rather than commenting out?

**Comment by willscott** [comment]
cleanup a comment, file an issue for reverting the error code. Otherwise good to merge.

File: spec/provider.integration.spec.js (hunk 1 above)
Comment (ryscheng) [inline spec/provider.integration.spec.js]: done

**Comment by ryscheng** [comment]
Done

**Pull request merged**